### PR TITLE
PODS-9835 - Remove unused configuration values

### PR DESCRIPTION
**Pull request by LauraToddCodes**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -56,22 +56,9 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientV2Module"
...
-  countdownSeconds = 120
-}
-
-nunjucks {
-  viewPaths = ["views"]
-  libPaths = ["govuk-frontend", "hmrc-frontend"]
-  noCache = false
-  routesBatchSize = 350
-  optimizationLevel = -1
-}
```
Comment (PavelVjalicin) [inline conf/application.conf:74]: Nunjucks is still used.
Leave the session config in for now.

File: conf/application.conf [hunk 2]
```diff
@@ -148,7 +135,6 @@ urls {
   govUkLink = "https://www.gov.uk/"
   contactHmrcLink = "https://www.gov.uk/government/organisations/hm-revenue-customs/contact/pension-scheme-enquiries"
   pensionAdministratorGovUkLink = "https://www.gov.uk/guidance/pension-administrators-register-as-an-administrator"
-  pensionPractitionerGovUkLink = "https://www.gov.uk/guidance/register-as-a-pension-scheme-practitioner"
```
Comment (PavelVjalicin) [inline conf/application.conf:138]: I wonder why this is not used? Maybe it should be?

File: conf/application.conf [hunk 3]
```diff
@@ -201,5 +186,3 @@ email.individualMigrationConfirmationTemplateId = "pods_racdac_individual_migrat
 email.force = false
 
 migration-data-cache.timeToLiveInDays = 28
-
-contact-frontend.serviceId = "PODS"
```
Comment (PavelVjalicin) [inline conf/application.conf:205]: Seems like something used by a library. Maybe has to do with feedback?

**Review by PavelVjalicin — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (LauraToddCodes) [reply conf/application.conf:74]: I'll put nunjucks back in but session values are duplicated within the same file

File: conf/application.conf (hunk 2 above)
Comment (LauraToddCodes) [reply conf/application.conf:138]: Maybe that only an administrator can migrate, not a practitioner?

File: conf/application.conf (hunk 1 above)
Comment (PavelVjalicin) [reply conf/application.conf]: Looks good.

**Pull request merged**